edge/pkg/common/dbm/db.go

```diff
@@ -35,7 +35,6 @@ func InitDBConfig(driverName, dbName, dataSource string) {
 		}()
 		// create orm
 		DBAccess = orm.NewOrmUsingDB(dbName)
-		klog.Infof("!!!!!!!!!!in db.go, DBAccess = %v", DBAccess)
```

Comment:
I do not quite understand this info message, how it could be useful to see Ormer object... i would also say we can just go ahead to remove this. or if necessary, perhaps we can change this into `debug` log at least.